library/core/src/main/java/com/google/android/exoplayer2/video/MediaCodecVideoRenderer.java

```diff
@@ -438,7 +438,7 @@ protected void onInputFormatChanged(Format newFormat) throws ExoPlaybackExceptio
   @Override
   protected void onQueueInputBuffer(DecoderInputBuffer buffer) {
     if (Util.SDK_INT < 23 && tunneling) {
-      maybeNotifyRenderedFirstFrame();
+      maybeNotifyRenderedFirstFrame(0);
```

Comment:
This needs to be correct also. I think you probably need `buffer.timeUs - streamOffsetUs`, where `streamOffsetUs` is somehow obtained from `BaseRenderer`. Would need to check this though.